Hardcodet.NotifyIcon.Wpf/Source/NotifyIconWpf/TaskbarIcon.cs

```diff
@@ -171,21 +175,21 @@ public Point GetPopupTrayPosition()
         /// <param name="balloon"></param>
         /// <param name="animation">An optional animation for the popup.</param>
         /// <param name="timeout">The time after which the popup is being closed.
-        /// Submit null in order to keep the balloon open inde
+        /// Submit null in order to keep the balloon open indefinitely
         /// </param>
         /// <exception cref="ArgumentNullException">If <paramref name="balloon"/>
         /// is a null reference.</exception>
         public void ShowCustomBalloon(UIElement balloon, PopupAnimation animation, int? timeout)
         {
-            Dispatcher dispatcher = this.GetDispatcher();
+            var dispatcher = this.GetDispatcher();
```

Comment:
Minor: That for example would be a case where I'd keep it (I don't know what type `GetDispatcher` returns. For all I know, it could return a Task...

Reply:
(Please don't feel you have to commit a correction for this, this is just fine!)

Reply:
I mildly disagree, a method called `GetDispatcher` should return a Dispatcher and not something which is a completely different thing. Naming is important!

But I have no issue rolling it back, it's your party :1st_place_medal: